lib/rspec/flakey_spec.rb

```diff
@@ -17,7 +17,11 @@ def initialize(example, attempts, retry_count)
     end
 
     def as_json
-      { attempts: @attempts, retry_count: @retry_count, location: @example.location, messages: messages }
+      { attempts:, retry_count:, location:, messages: }
+    end
+
+    def location
+      @example.location
```

Comment:
Should this be private?  If not should we give it an attr-reader and set it in the initializer?